invenio_rdm_records/config.py

```diff
@@ -479,3 +481,17 @@ def make_doi(prefix, record):
         ...
     }]
 """
+
+# Feature flag to enable/disable Resource access tokens (RATs)
+RDM_RESOURCE_ACCESS_TOKENS_ENABLED = True
```

Comment:
```suggestion
RDM_RESOURCE_ACCESS_TOKENS_ENABLED = False
```

minor: maybe disable by default (it also looks like this is the assumption from the tests)